### ISSUE TITLE
Receive news updates

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#31]
As a primary user, I can see the latest platform and programme news, so that I'm always up to date with new features and announcements.

This should be integrated with user notifications #26.

**Comment by asimonw** [comment]
![fullsizerender_4](https://cloud.githubusercontent.com/assets/15235804/17745626/7b2edb02-64a4-11e6-99c4-03acd2b66874.jpg)

A simple list of news items, with title, date/time and text (might include links), no images or other media at this point. This list could also contain automated messages (e.g. new member joining) or even personalised ones (new match was made for your challenge, ...). To be further explored in #26. This issue is only about the news sent out by admins.

**Comment by JMurphyWeb** [comment]
@asimonw here is a view for admin view news page:

![screen shot 2016-11-14 at 12 04 03](https://cloud.githubusercontent.com/assets/12638261/20264280/9fca9ef2-aa62-11e6-8adb-a21a111ba950.png)


Are archived news items shown? if so how does an admin know if they are archived or not?

**Comment by asimonw** [comment]
@JMurphyWeb Good point, archived items should be shown (how else would an admin be able to access them?), but we should indicate which ones are "Active" and which ones are not. In our mockups for "People" we do this with green and red dots (something which would be nice to implement). I'd suggest using that for News as well. What do you think @fannyBkane ?

By the way, this is currently a problem for orgs as well. We should use the same solution across the board. I'll create a separate issue for orgs.

**Comment by fannyBkane** [comment]
I agree with using the red and green dots on admin views (that includes lists of People, News, Organisations, and Insights & resources) to show what items are enabled or not.

**Comment by JMurphyWeb** [comment]
Sounds good. Thanks @asimonw and @fannyBkane. 

Said issue is #334
